doc/manual/50-feature-details.en.md

```diff
@@ -41,15 +38,15 @@ relevant savings in disk usage. This is particularly true of VLDB
 contexts and of those databases containing a high percentage of
 _read-only historical tables_.
 
-Barman implements incremental backup through a global/server option
+Rsync incremental backups can be enabled through a global/server option
 called `reuse_backup`, that transparently manages the `barman backup`
 command. It accepts three values:
 
 - `off`: standard full backup (default)
-- `link`: incremental backup, by reusing the last backup for a server
+- `link`: file-level incremental backup, by reusing the last backup for a server
   and creating a hard link of the unchanged files (for backup space
   and time reduction)
-- `copy`: incremental backup, by reusing the last backup for a server
+- `copy`: file-level incremental backup, by reusing the last backup for a server
```

Comment:
I wonder that we can add a note after this explaining that users of `rsync` don't need to take full and incremental backups, i.e. they can take all backups with `reuse_backup = link` if they want, and the first backup in the catalog will copy all files while the 2nd onwards only the deduplicated files.

With this, I'm willing to avoid a recurrent situation where people schedule some backups with `off` and others with `link` e.g.

Reply:
Just added a note about that at the end of the Rsync section.